trio/tests/test_ssl.py

```diff
@@ -98,14 +100,20 @@ def ssl_echo_serve_sync(sock, *, expect_fail=False):
                     # respond in kind but it's legal for them to have already
                     # gone away.
                     exceptions = (BrokenPipeError, ssl.SSLZeroReturnError)
-                    # Under unclear conditions, CPython sometimes raises
-                    # SSLWantWriteError here. This is a bug (bpo-32219), but
-                    # it's not our bug, so ignore it.
-                    exceptions += (ssl.SSLWantWriteError,)
                     try:
                         wrapped.unwrap()
                     except exceptions:
                         pass
+                    except ssl.SSLWantWriteError:  # pragma: no cover
+                        # Under unclear conditions, CPython sometimes raises
+                        # SSLWantWriteError here. This is a bug (bpo-32219),
+                        # but it's not our bug, so ignore it.
```

Comment:
Maybe mention why it is only being complained about for 3.8+ since "ignore it" is no longer true.